understanding/20/audio-description-prerecorded.html

```diff
@@ -186,7 +186,11 @@ <h3>Sufficient Techniques for Audio Description (Prerecorded)</h3>
                      <a href="../Techniques/smil/SM7" class="smil">Providing audio description in SMIL 2.0</a>
                      										           
                   </li>
-                  
+
+                  <li>
+                     <a href="../Techniques/general/G226" class="general">Providing audio descriptions by incorporating narration in the existing soundtrack</a>
```

Comment:
```suggestion
                     <a href="../Techniques/general/G226" class="general">Providing audio descriptions by incorporating narration in the main soundtrack</a>
```

I guess "existing" doesn't apply to new videos?

Reply:
Yeah, this is challenging. The reason I shied away from using "main" -- which I agree is logical -- is because of the wording of the audio description definition:
> narration added to the soundtrack to describe important visual details that cannot be understood from the main soundtrack alone

In a perfect world, I'd alter this definition to strip out the final phrase, and just make it "Narration added to the soundtrack to describe important visual details." That is a really nice definition, which gives us reasonable  flexibility. 

But despite all the qualifying notes, this is the normative wording and it states pretty clearly that it is in _addition_ to the main soundtrack. So I thought it was a little contradictory to say we are incorporating something in the "main soundtrack".

My use of "existing" isn't ideal, but the idea is that this is being remediated during production as opposed to post-production. Does "current" work any better?

Another possible approach is to strip out any adjective and say "soundtrack"?

Reply:
I've removed "existing", as suggested.